service/tests/resources/not-shuttle/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+/// This service cannot be hosted on shuttle since it is missing the entrypoint the shutlle macros would have added!!!
```

Comment:
Any reason why we have all the wait code in there? Is this not equivalent as a test to if this was empty?